### PR TITLE
Add Anonaddy

### PR DESCRIPTION
**Pull request by person1234-ux**
Was having trouble with the svg icon so you should probably take a look at that before your merge it.

### CODE REVIEW

File: img/a/anonaddy.com.svg [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                <path id="XMLID_76_" class="st1" d="m1651.9 195.3c0-40.4 20.8-62.1 62.1-62.1 9.2 0 20.8 1.2 35.4 3.9v-56.3c0-6.8 2.1-9.2 8.9-9.2h16.4c6.8 0 9.2 2.4 9.2 9.2v189.4c0 9.5-2.4 12.5-11.9 16.4-16.9 6.2-35.4 9.5-55.6 9.5-43.1 0-64.5-19.9-64.5-60.1zm97.5-26.7c-11.9-2.7-21.4-3.9-28.8-3.9-22.9 0-34.2 10.4-34.2 31.5v38.7c0 19.9 11 29.7 33.3 29.7 10.4 0 19.6-1.2 27.4-3.3 1.5-0.6 2.4-1.8 2.4-3.3v-89.4z" style="fill:#f5f7fa"/>
+                <path id="XMLID_79_" class="st1" d="m1939.1 137.6c6.8 0 9.2 2.1 9.2 8.9v150.2c0 40.7-20.5 64.8-65.4 64.8-29.4 0-50.3-9.5-61.8-28.5-4.2-6.5-3-11.6 3.9-14.9l11.3-6.2c7.1-3.6 10.4-2.7 14.9 3.6 6.2 10.1 16.6 15.2 31.2 15.2 21.1 0 31.5-10.7 31.5-32.1v-14c-12.2 2.7-24.1 4.2-35.4 4.2-41.6 0-62.1-21.4-62.1-59.8v-82.4c0-6.8 2.4-8.9 9.2-8.9h16.4c6.8 0 8.9 2.1 8.9 8.9v81.2c0 19.6 10.7 29.4 32.4 29.4 9.8 0 19.9-1.5 30.6-4.2v-106.5c0-6.8 2.4-8.9 9.2-8.9z" style="fill:#f5f7fa"/>
+		</g>
+	</g>
+</g>
+</svg>
```
Comment (Carlgo11) [inline img/a/anonaddy.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 69.77 47.45"><path fill="#3ae7e1" d="M8.75 21.66a2.08 2.08 0 0 1-1.6-3.69l5.75-4.5a2.08 2.08 0 0 1 2.56 3.27l-5.76 4.5c-.28.23-.61.37-.95.42zM2.4 37.87a2.08 2.08 0 0 1-1.6-3.7l12.65-9.86A2.08 2.08 0 0 1 16 27.58L3.36 37.46c-.3.22-.62.36-.95.4zm16.9-1.95a2.08 2.08 0 0 1-1.6-3.69l10.38-8.1a2.08 2.08 0 0 1 2.56 3.28l-10.37 8.1c-.28.23-.6.37-.95.41zM65.92.01H17.84c-1.8 0-3.46 1.1-4 2.8a4 4 0 0 0 1.36 4.3s18.43 14.3 22.94 17.42a3.5 3.5 0 0 0 4.13-.04L65.64 6.31v31.43a5.56 5.56 0 0 1-5.56 5.56H15.9c-1.14 0-2.06.92-2.06 2.06a2.1 2.1 0 0 0 2.09 2.09h44.13a9.7 9.7 0 0 0 9.7-9.7V3.84A3.83 3.83 0 0 0 65.9 0zM40.18 20.87 18.78 4.14h42.9z"/><path fill="#f5f7fa" d="M3.16 26.05a2.1 2.1 0 0 1-1.97-.76c-.7-.9-.54-2.2.36-2.91l.55-.42a2.08 2.08 0 0 1 2.55 3.27l-.54.42a2.1 2.1 0 0 1-.95.4zm17.26-2.19a2.08 2.08 0 0 1-1.6-3.7l2-1.57a2.08 2.08 0 0 1 2.56 3.27l-2 1.58c-.3.21-.63.35-.96.42zm-6.77 16.56a2.08 2.08 0 0 1-1.6-3.69l.73-.57a2.08 2.08 0 0 1 2.55 3.27l-.73.57a2 2 0 0 1-.95.42z"/></svg>
```

File: entries/a/anonaddy.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Anonaddy": {
+    "domain": "anonaddy.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://anonaddy.com/security/",
```
Comment (Carlgo11) [inline entries/a/anonaddy.com.json:8]: The documentation element is meant to explain to users how they can enable 2FA on the service. This link doesn't explain that. Could you please find another, more relevant, link? If not, please remove this line.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/a/anonaddy.com.json (hunk 2 above)
Comment (person1234-ux) [reply entries/a/anonaddy.com.json:8]: github doesn't to like me very much and is error out when I try to access my branch. let me see what I can do it access it. I am going to remove the documentation

Comment (Carlgo11) [reply entries/a/anonaddy.com.json:8]: Hmm yes I also get the 500 error. Maybe wait an hour or so? If it doesn't work then, you might want to consider removing that branch and creating a new PR.

Comment (Carlgo11) [reply entries/a/anonaddy.com.json:8]: Maybe this works :shrug: 
```suggestion
```

**Comment by person1234-ux** [comment]
sorry  for the ping still having that 500 issue I contacted github support to see what they can do as I can't even clone fork I created

**Comment by Carlgo11** [comment]
It's possible to clone the code this way:
```SH
# Add reference to main repo
git remote add upstream git@github.com:2factorauth/twofactorauth.git

# Fetch the code from this PR
git fetch upstream +refs/pull/6708/merge:

# Check out the fetched code
git checkout -qf FETCH_HEAD

# Create new branch with the code
git checkout -b adding-anonaddy
```

**Pull request closed without merging**